helmcharts/services/kube-prometheus-stack/charts/grafana/values.yaml

```diff
@@ -1399,4 +1399,4 @@ extraObjects: []
 #
 # Alternatively, if you wish to allow secret values to be exposed in the rendered grafana.ini configmap,
 # you can disable this check by setting assertNoLeakedSecrets to false.
-assertNoLeakedSecrets: true
+assertNoLeakedSecrets: false
```

Comment:
Is there any reason to turn it off?

Reply:
yeah  throwing error `Sensitive key 'smtp.password' should not be defined explicitly in values `during deployment thats why i removed